[2022-12-14T12:21:31.280+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: weekly_wrapped_dag.generate_wrapped manual__2022-12-14T12:21:28.693428+00:00 [queued]>
[2022-12-14T12:21:31.288+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: weekly_wrapped_dag.generate_wrapped manual__2022-12-14T12:21:28.693428+00:00 [queued]>
[2022-12-14T12:21:31.288+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-14T12:21:31.289+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-14T12:21:31.290+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-14T12:21:31.299+0000] {taskinstance.py:1304} INFO - Executing <Task(_PythonDecoratedOperator): generate_wrapped> on 2022-12-14 12:21:28.693428+00:00
[2022-12-14T12:21:31.307+0000] {standard_task_runner.py:55} INFO - Started process 27300 to run task
[2022-12-14T12:21:31.313+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weekly_wrapped_dag', 'generate_wrapped', 'manual__2022-12-14T12:21:28.693428+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/weekly_dag.py', '--cfg-path', '/tmp/tmpcabe36xj']
[2022-12-14T12:21:31.315+0000] {standard_task_runner.py:83} INFO - Job 76: Subtask generate_wrapped
[2022-12-14T12:21:31.406+0000] {task_command.py:389} INFO - Running <TaskInstance: weekly_wrapped_dag.generate_wrapped manual__2022-12-14T12:21:28.693428+00:00 [running]> on host b92abc15fbab
[2022-12-14T12:21:31.511+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weekly_wrapped_dag
AIRFLOW_CTX_TASK_ID=generate_wrapped
AIRFLOW_CTX_EXECUTION_DATE=2022-12-14T12:21:28.693428+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-14T12:21:28.693428+00:00
[2022-12-14T12:21:31.531+0000] {base.py:73} INFO - Using connection ID 'spotify_etl' for task execution.
[2022-12-14T12:21:31.544+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 217, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/weekly_dag.py", line 47, in generate_wrapped
    print(json.loads(result))
  File "/usr/local/lib/python3.7/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not list
[2022-12-14T12:21:31.571+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=weekly_wrapped_dag, task_id=generate_wrapped, execution_date=20221214T122128, start_date=20221214T122131, end_date=20221214T122131
[2022-12-14T12:21:31.611+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 76 for task generate_wrapped (the JSON object must be str, bytes or bytearray, not list; 27300)
[2022-12-14T12:21:31.648+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-14T12:21:31.666+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
